actors/builtin/miner/miner_actor.go

```diff
@@ -1870,22 +1946,34 @@ func qaPowerForSectors(sectorSize abi.SectorSize, sectors []*SectorOnChainInfo)
 }
 
 func unlockDeclaredFaultPenalty(st *State, store adt.Store, currEpoch abi.ChainEpoch, epochTargetReward abi.TokenAmount, networkQAPower abi.StoragePower, sectors []*SectorOnChainInfo) (abi.TokenAmount, error) {
-	sectorSize := st.Info.SectorSize
+	info, err := st.GetInfo(store)
+	if err != nil {
+		return abi.NewTokenAmount(0), err
+	}
+	sectorSize := info.SectorSize
```

Comment:
Just pass `info.SectorSize` into `qaPowerForSectors`. This variable isn't necessary.

Reply:
Here and in the next two functions.

Reply:
The call sites have already loaded the info, so pass the sector size as a parameter to `unlockXxxPenalty`